### ISSUE TITLE
Function is fitted to strip transect

### ISSUE THREAD

**Issue by erex** [DistanceDevelopment/Distance#57]
From [list on 22Apr20](https://groups.google.com/forum/#!topic/distance-sampling/dnHbt2frCAk)
```
# Generate half normal detection function
df_hn <- ds(
  data = observations,
  cutpoints = c(0,100),
  key = "hn")
```
Produces
```
Summary for distance analysis 
Number of observations :  17 
Distance range         :  0  -  100 

Model : Half-normal key function 
AIC   : 2 

Detection function parameters
Scale coefficient(s):  
            estimate    se
(Intercept) 3.914021 1e+05

                      Estimate         SE       CV
Average p            0.5990685   46265.17 77228.51
N in covered region 28.3773873 2191543.41 77228.51
```

Clearly the estimates of sigma and hence P_a is incorrect.  Shouldn't a warning/error be thrown?

**Comment by lenthomas** [comment]
BTW, I wondered what the poster might have input as data, so I tried
`observations <- rep(100, 17)`
and got his results; but then I tried
`observations <- rep(0, 17)`
and got the same results.  Cool!

My feeling in general is that we can't check for every clearly incorrect type of dataset that users might want to throw at the software.  So one option might be to leave as is, especially since there are higher priorities for developer time.  On the other hand, if we do want to check, then what should we check for?  

One option is to check for all distance values being the same, and throw an error.  I did wonder about allowing this though for users who want to try to analyze strip transect data with `Distance`/`mrds` - but there's probably a better way to achieve this.

Another option would be to check for a large CV, but then the question of what is large, and in any case, if people check the CV they can see it's large!
